.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.1.0
```

Comment:
Could you please update to v4.2.1 and pin node versions, it should look sth like:

```
    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.2.1
    with:
      lint: true
      license-check: true
      node-versions: '["16", "18", "20", "22"]'
```

Reply:
Done it